docs/source/quickstart.rst

```diff
@@ -0,0 +1,60 @@
+Quickstart
+==========
+
+Reading
+-------
+
+To open and read the content of a parquet file:
+
+.. code-block:: python
+
+    from fastparquet import ParquetFile
+    pf = ParquetFile('myfile.parq')
+    df = pf.to_pandas()
+
+The pandas data-frame, `df` will contain all columns in the target file, and all row-groups concatenated together. If the data is a multi-file collection, such as generated by hadoop, the filename to supply is
+either the directory name, or the "_metadata" file contained therein - these are handled transparently.
+
+One may wish to investigate the meta-data associated with the data before loading, e.g., to choose which row-groups and columns to load. The properties `columns`, `count`, `dtypes` and `statistics` are available
```

Comment:
The docs people get upset with me when I use "e.g." instead of "for example" because apparently it isn't clear to non-native speakers.  No strong thoughts here, just passing it along.